app/javascript/components/Pages/EmotionEntry.js

```diff
@@ -125,13 +126,15 @@ const EmotionEntry = ({data, setData, saveDataToDb, steps, service}) => {
             <Emojis show={show}/>
           </CSSTransition>
         </div>
-        <div className='d-flex justify-content-between m-3'>
-          <ShoutOutIcon/>
+        <div className='d-flex position-absolute placement-buttons justify-content-between col-6 offset-3'>
```

Comment:
are you sure **col-6 offset-3** is useful here? I do not see the **row** class above. Maybe **w-50** could work better here?

Reply:
Yes of course this is work in this case very well and I think this decision more preferable in this case.